deno.json

```diff
@@ -24,10 +24,13 @@
     "typos": "typos -c ./.github/workflows/typos.toml",
     "build:crypto": "deno task --cwd crypto/_wasm wasmbuild",
     "wasmbuild": "deno run -A jsr:@deno/wasmbuild@0.17.1 --js-ext mjs --sync",
-    "cov": "deno coverage --ignore=\"**/*.generated.mjs,**/_test_utils.ts,.github\"",
+    "cov": "deno task test --coverage --clean --reporter=dot && deno coverage --ignore=\"**/*.generated.mjs,**/_test_utils.ts,.github\"",
```

Comment:
This seems causing the entire test cases run multiple times in CI. Can you also modify ci.yml to fix that?